source/MaterialXContrib/MaterialXNode/Plugin.cpp

```diff
@@ -22,6 +22,8 @@ void Plugin::initialize(const std::string& loadPath)
 	MaterialX::FilePath searchPath(loadPath);
 	_librarySearchPath = searchPath / MaterialX::FilePath("../../libraries");
 	_resourcePath = searchPath / MaterialX::FilePath("../resources");
+    // Set to resource path for now
```

Comment:
This is still fixed but just not a hard-coded path so each of us doesn't have to modify the code.